src/pickup.cpp

```diff
@@ -169,6 +181,44 @@ interact_results interact_with_vehicle( vehicle *veh, const tripoint &pos,
             veh_tool( "hotplate" );
             return DONE;
 
+        case USE_WASHMACHINE: {
+            bool detergent_is_enough = g->u.crafting_inventory().has_charges( "detergent", 5 );
```

Comment:
Speaking of code, I'd turn the whole snippet into a named function to prevent further bloating of `interact_with_vehicle`.

Reply:
Fixed in 8cc9d3f.